lighthouse-plugin-ad-speed-insights/plugin.js

```diff
@@ -54,6 +55,7 @@ module.exports = {
       {id: 'ad-blocking-tasks', weight: 1, group: 'ads-performance'},
       {id: 'ad-request-critical-path', weight: 1, group: 'ads-performance'},
       {id: 'serial-header-bidding', weight: 1, group: 'ads-performance'},
+      {id: 'idle-network-times', weight: 0.5, group: 'ads-performance'},
```

Comment:
What are your thoughts on splitting this into a new experimental group? The group could be displayed based on a `--experimental` flag.

Reply:
SGTM, but I'll defer it for a future PR. It's also worth having a broader discussion around weighing audits, etc.

Reply:
Could you put up an issue or add a TODO?